src/app/components/transactionSetting/editBtcFee.tsx

```diff
@@ -0,0 +1,421 @@
+import useDebounce from '@hooks/useDebounce';
+import useOrdinalsByAddress from '@hooks/useOrdinalsByAddress';
+import useWalletSelector from '@hooks/useWalletSelector';
+import { Faders } from '@phosphor-icons/react';
+import {
+  currencySymbolMap,
+  ErrorCodes,
+  getBtcFees,
+  getBtcFeesForNonOrdinalBtcSend,
+  getBtcFeesForOrdinalSend,
+  getBtcFiatEquivalent,
+  Recipient,
+  UTXO,
+} from '@secretkeylabs/xverse-core';
+import { StyledP } from '@ui-library/common.styled';
+import BigNumber from 'bignumber.js';
+import { useEffect, useMemo, useRef, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { NumericFormat } from 'react-number-format';
+import styled from 'styled-components';
+import Theme from 'theme';
+import FeeItem from './feeItem';
+
+const Container = styled.div((props) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  marginLeft: props.theme.space.m,
+  marginRight: props.theme.space.m,
+  paddingBottom: props.theme.space.m,
+}));
+
+const DetailText = styled.h1((props) => ({
+  ...props.theme.typography.body_m,
+  color: props.theme.colors.white_200,
+}));
+
+interface InputContainerProps {
+  withError?: boolean;
+}
+const InputContainer = styled.div<InputContainerProps>((props) => ({
+  display: 'flex',
+  flexDirection: 'row',
+  alignItems: 'center',
+  justifyContent: 'space-between',
+  marginTop: props.theme.space.xs,
+  marginBottom: props.theme.space.s,
+  border: `1px solid ${
+    props.withError ? props.theme.colors.feedback.error : props.theme.colors.elevation6
+  }`,
+  backgroundColor: props.theme.colors.elevation1,
+  borderRadius: props.theme.radius(1),
+  padding: props.theme.space.s,
+}));
+
+const InputField = styled.input((props) => ({
+  ...props.theme.body_m,
+  backgroundColor: 'transparent',
+  color: props.theme.colors.white_0,
+  border: 'transparent',
+  width: '50%',
+  '&::-webkit-outer-spin-button': {
+    '-webkit-appearance': 'none',
+    margin: 0,
+  },
+  '&::-webkit-inner-spin-button': {
+    '-webkit-appearance': 'none',
+    margin: 0,
+  },
+  '&[type=number]': {
+    '-moz-appearance': 'textfield',
+  },
+}));
+
+const FeeText = styled.h1((props) => ({
+  ...props.theme.typography.body_m,
+  color: props.theme.colors.white_0,
+}));
+
+const FeeContainer = styled.div({
+  display: 'flex',
+  flexDirection: 'column',
+});
+
+const ErrorText = styled.h1((props) => ({
+  ...props.theme.typography.body_s,
+  color: props.theme.colors.danger_light,
+  marginBottom: props.theme.space.xxs,
+}));
+
+const FeePrioritiesContainer = styled.div`
+  display: flex;
+  margin-top: ${(props) => props.theme.space.m};
+  flex-direction: column;
+`;
+
+interface FeeContainerProps {
+  isSelected: boolean;
+}
+
+const FeeItemContainer = styled.button<FeeContainerProps>`
+  display: flex;
+  padding: 12px 16px;
+  align-items: center;
+  gap: 12px;
+  align-self: stretch;
+  border-radius: 12px;
+  border: 1px solid var(--White-850, rgba(255, 255, 255, 0.15));
```

Comment:
Are we planning on using css variables like `--White-850` in the near future? As I can see, we don't have any yet